calico-vpp-agent/policy/policy_server.go

```diff
@@ -84,7 +84,9 @@ type Server struct {
 	workloadsToHostPolicy *Policy
 	workloadAddresses     map[string]string
 	/* always allow traffic coming from host to the pods (for healthchecks and so on) */
-	allowFromHostPolicyId uint32
+	IngressAllowFromHostPolicyId uint32
+	EgressAllowFromHostPolicyId  uint32
+	AllowFromHostIPSet           *IPSet
```

Comment:
Same thing here, we should probably add a comment stating that this is the ipset for the AllowFromHostPolicy
````golang 
// AllowFromHostIPSet
AllowFromHostIPSet *IPSet
````